docs/install_guides/installing-model-dev-tools.md

```diff
@@ -91,65 +91,67 @@ Note that the commands are different for a Python installation and a C++ install
-      
+
 .. tab:: C++
 
-  When using OpenVINO Development Tools for C++ development, it’s important to install the same version as OpenVINO Runtime. Following the instructions below will ensure that you are installing a version that matches that of OpenVINO Runtime.
-  
-  **Recommended: Install Using the Requirements Files**
-  After you have installed OpenVINO Runtime from an archive file, you can find a set of requirements files in the <INSTALL_DIR>\tools\ directory. The requirements files will install the matching version of OpenVINO Development Tools and its dependencies.
-  
-  1. Install the OpenVINO Development Tools mandatory requirements using the following command:
+   When using OpenVINO Development Tools for C++ development, it’s important to install the same version as OpenVINO Runtime. Following the instructions below will ensure that you are installing a version that matches that of OpenVINO Runtime.
+
+   **Recommended: Install Using the Requirements Files**
+
+   After you have installed OpenVINO Runtime from an archive file, you can find a set of requirements files in the <INSTALL_DIR>\tools\ directory. The requirements files will install the matching version of OpenVINO Development Tools and its dependencies.
+
+   1. Install the OpenVINO Development Tools mandatory requirements using the following command:
+      .. code-block:: sh
+
+         pip install -r <INSTALL_DIR>\tools\requirements.txt
+
+   2. If you are using additional frameworks, you must also install the requirements for those frameworks using the corresponding requirements file. For example, if you are using a TensorFlow model, use the following command to install requirements for TensorFlow:
+      .. code-block:: sh
+
+         pip install -r <INSTALL_DIR>\tools\requirements_tensorflow2.txt
+
+   **Alternative: Install the openvino-dev Package from PyPI**
+
+   You can also install OpenVINO Development Tools from PyPI using the following command.
 
-   .. code-block:: sh
-   
-      pip install -r <INSTALL_DIR>\tools\requirements.txt
-      
-  2. If you are using additional frameworks, you must also install the requirements for those frameworks using the corresponding requirements file. For example, if you are using a TensorFlow model, use the following command to install requirements for TensorFlow:
-     
-   .. code-block:: sh
-   
-      pip install -r <INSTALL_DIR>\tools\requirements_tensorflow2.txt
-  
-  **Alternative: Install the openvino-dev Package from PyPI**
-  You can also install OpenVINO Development Tools from PyPI using the following command.
-  
    .. important::
-    Make sure to specify the `openvino-dev` version that matches your installed version of OpenVINO Runtime. Otherwise, compatibility errors are likely to occur.
-    
+
+      Make sure to specify the `openvino-dev` version that matches your installed version of OpenVINO Runtime. Otherwise, compatibility errors are likely to occur.
```

Comment:
```suggestion
      Make sure that the `openvino-dev` version you specified matches your installed version of OpenVINO Runtime. Otherwise, compatibility errors are likely to occur.
```